### DISCUSSION TITLE
Queue Size Limited?

### DISCUSSION THREAD

**Discussion by Brat-Wurscht** [CappielloAntonio/tempo#242] [Q&A]
Hi there!
Whenever I choose to play a larger playlist, the resulting queue list has never more than 100 entries. Is this a setting I can adjust somewhere? (I haven't found it)

Android App Version 3.8.1 (25)

**Comment by skarrok** [comment]
Yep, same issue here.

Can't use the app with large playlists.

**Reply by skarrok** [reply]
Found it in code
Here https://github.com/CappielloAntonio/tempo/blob/main/app/src/main/java/com/cappielloantonio/tempo/ui/fragment/PlaylistPageFragment.java#L164
and here
https://github.com/CappielloAntonio/tempo/blob/main/app/src/main/java/com/cappielloantonio/tempo/ui/fragment/PlaylistPageFragment.java#L170

playing or shuffling playlist limited to 100 songs

intoroduced in this commit https://github.com/CappielloAntonio/tempo/commit/623a4956a5a3765f151db540ed64925211748a1c

**Reply by Brat-Wurscht** [reply]
That's very interesting. Thanks for sharing this.

I wonder why this restriction was built in. If the dev / one of the devs could give us some feedback here on occasion, that would be great.

This is a noticeable limitation for my use of an otherwise excellent app.

**Reply by skarrok** [reply]
Created issue https://github.com/CappielloAntonio/tempo/issues/258

Also tried to revert this commit and test on my phone.
Without 100 song limit playing or shuffling playlist works as expected, but opening play queue with 500 songs freezes UI for a couple of seconds. After such long opening play queue scrolls fine until closed and with next reopening freezes UI again.

I think I'm gonna use Tempo like this (I don't open play queue often) until properly fixed by the author.

**Comment by shrapnelnet** [comment]
#385
